native/src/main/java/io/ballerina/stdlib/os/utils/OSUtils.java

```diff
@@ -35,15 +35,12 @@ private OSUtils() {
      * @return os property as a {@link String} or {@code PredefinedTypes.TYPE_STRING.getZeroValue()} if the
      * property does not exist.
      */
-    public static String getSystemProperty(String key) {
-        String value = System.getProperty(key);
+    public static BString getSystemProperty(BString key) {
```

Comment:
is this native function used by some other module

Reply:
It seems this is not used by other modules.